### ISSUE TITLE
No contact export or import

### ISSUE THREAD

**Issue by Ieubts** [owncloud/contacts#310]
### Steps to reproduce

1.
2.
3.
### Expected behaviour

Tell us what should happen
### Actual behaviour

Tell us what happens instead
### Server configuration

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**ownCloud version:** (see ownCloud admin page)

**Contacts version:** (see ownCloud apps page)

**Updated from an older ownCloud or fresh install:**

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```

**The content of config/config.php:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your ownCloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
### Client configuration

**Browser:**

**Operating system:**

**CardDAV-clients:**
#### Web server error log

```
Insert your webserver log here
```
#### ownCloud log (data/owncloud.log)

```
Insert your ownCloud log here
```
#### Browser log

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by aspdye** [comment]
Can you give a little description of the enhancement, or fill out the issue template :wink:

**Comment by aspdye** [comment]
Dup of many other issues :stuck_out_tongue_winking_eye:

**Comment by Ieubts** [comment]
Thank you.  I'm new to Owncloud and I guess I really made a mistake with upgrading.  I converted to using the contacts app (worked great) and now after I upgraded today  to 9.0... and I guess contacts 1.1?.... most of the functionality is gone.  Can you point me to where I can at least read about what has happened? (a forum?).  I need to roll back as now I can't import/export, delete multiple contacts, can't group, etc. 

```
On Monday, March 28, 2016 3:15 PM, Thomas Müller <notifications@github.com> wrote:
```

 Closed #310.—

**Comment by DeepDiver1975** [comment]
- Import of a single contact is possible in the settings menu
- Export needs to be implemented
- multiple delete needs to be implemented
- groups can be managed within each contact

Feel free to open a new issue for multi delete - THX
